src/tests/Microsoft.PowerFx.Interpreter.Tests/DateTimeTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Interpreter.Tests
+{
+    public class DateTimeTests
+    {
+        private static readonly bool IsLocalUTC = TimeZoneInfo.Local.BaseUtcOffset == TimeSpan.Zero;
+
+        [Fact]
+        public async void DateTimeUTCTests()
```

Comment:
>DateTimeUTCTests [](http://example.com/codeflow?start=26&length=16)

Nice!

Can we add a test that does a DateDiff between 2 dates, one in locale, one in UTC?

Reply:
Added